client/packages.go

```diff
@@ -96,10 +102,16 @@ func (client *Client) ListSnaps(names []string) ([]*Snap, error) {
 
 // FindSnaps returns a list of snaps available for install from the
 // store for this system and that match the query
-func (client *Client) FindSnaps(query string) ([]*Snap, *ResultInfo, error) {
-	q := url.Values{}
+func (client *Client) FindSnaps(opts *FindOptions) ([]*Snap, *ResultInfo, error) {
```

Comment:
s/FindSnaps/Find/

We should change List to become similar too (ListOptions, and ResultInfo), but we can do that independently.